gradle/scripts/nexusPublishing.gradle

```diff
@@ -64,6 +64,74 @@ gradle.taskGraph.afterTask {
     }
 }
 
+ext.pomAttributes = {
```

Comment:
Can we update and reuse the pomAttributes from bintrayPublishing.gradle? Idea is to have just one location for pom attributes